layouts/partials/topnav.html

```diff
@@ -15,15 +15,15 @@
     </div>
     <div class="buttons">
       <div class="search-btn">
-        <button id="search-btn"
-          onclick="toggleSidebar('sidebar-open');document.getElementById('algolia-search-input').focus();return false;"><span
-            class="cf-icon Search_New"></span></button>
```

Comment:
This shouldn't be removed or at least it needs to be refactored. When the sidebar is closed, a search icon appears in the top nav next to the gear icon. When a user clicks the search icon, this opens the sidebar and focuses the search text field.

Reply:
Oof, not sure why I deleted that. Sorry

Reply:
Restored and refactored into module.